src/styles/_stars.scss

```
@@ -0,0 +1,33 @@
+@use './mixins' as *;
+@use './variables' as *;
+@use './card' as *;
+
+.stars {
+  @include center;
+
+  line-height: 14px;
+  margin-top: 16px;
+  width: $primary-width;
```

width should be 100% so it would adapt if card change it's width 
```suggestion
  width: 100%;
```